cli/src/main/java/io/specmesh/cli/Provision.java

```diff
@@ -29,26 +27,25 @@
 import java.util.Map;
 import java.util.Properties;
 import java.util.concurrent.Callable;
-import lombok.Builder;
-import lombok.Getter;
-import lombok.experimental.Accessors;
 import picocli.CommandLine;
 import picocli.CommandLine.Option;
 
 /** SpecMesh Kafka Provisioner */
+@SuppressWarnings("unused")
 @Command(
         name = "provision",
         description =
                 "Apply a specification.yaml to provision kafka resources on a cluster.\n"
                         + "Use 'provision.properties' for common arguments\n"
                         + " Explicit properties file location /app/provision.properties\n\n")
-@Getter
-@Accessors(fluent = true)
-@Builder
-@SuppressFBWarnings
-public class Provision implements Callable<Integer> {
+public final class Provision implements Callable<Integer> {
```

Comment:
`Provision` type is no longer using the builder pattern. Instead, the `Provisioner` is a builder and this class is just there to handle the command line arguments.